golden_response/staging/get_pop_obs/moscow.json

```diff
@@ -242,82 +242,6 @@
         }
       ]
     },
-    "dc/p/119gy3zecnm61": {
-      "observations": [
```

Comment:
Confirmed via `grep -ri "moscow" REGION_DEMOGR_death_tl3.csv` returning empty that this was a good diff.

Reply:
Russia only reported country level for this dataset.